en/PlanView/PlanView.md

```diff
@@ -57,12 +52,35 @@ On the right edge of the display is the list of mission commands for this missio
 
 Click on a mission command to show its editor which allows you to specify the values for the command. You can also change the type of the command by clicking on the command name, "Waypoint" in this example. This allows you to pick from the set up available commands to build your mission. To the right of the command name is a menu you can open by clicking. This menu provides you access to additional options such as Insert and Delete.
 
+### Mission Settings
+The Mission Settings panel allows you to specify values which apply to the entire mission, or settings you want to control right at the beginning of a mission. This is the first item in the mission list on the right of the screen. For example, you can change the altitude for all mission waypoints to the value you set in "Waypoint alt".
+
+<img src="../../images/plan/MissionSettings.png" style="width: 200px;"/>
+
+### Repeat Mission
```

Comment:
I'm not sure why there is a need to call this out specifically:
* First, it is used almost never
* Second, there are many other commands available beyond Jump to Item

Reply:
I didn't know about this command until my friend wanted to use it and asked me if there was any command to repeat missions. 

Since I couldn't find any documentation related to this command I added it here. 
You are right. Maybe I should add a list of all other commands and their one line descriptions here?

Reply:
> Maybe I should add a list of all other commands and their one line descriptions here?

What I would instead do is explain how to change a command and get the list of all available commands from the ui:
* Click command name
* Select "All Commands" from drop down

The list of available commands is different based on firmware and vehicle type. And it also changes as firmwares add new features. Would be better to not have to chase a moving target in the docs.

Reply:
Yes, either exhaustive or refer to other source for info (e.g. UI in this case). 

What you could do here though is list a few examples - ie
* Select "All Commands" from drop down
  > **Note** The list of available commands will depend on firmware and vehicle type. Examples may include waypoints, camera trigger, repeat etc.